extensions/css-language-features/package.json

```diff
@@ -356,6 +360,28 @@
             "scope": "resource",
             "default": null,
             "markdownDescription": "%css.format.maxPreserveNewLines.desc%"
+          },
+          "css.format.formatPastedFiles": {
```

Comment:
Instead of `css.format`, maybe put these under `css.paste.*` and `css.drop.*`. @aeschli  Can also help with better suggestions for where these should live